src/main/java/org/odpi/openmetadata/adapters/connectors/integration/eventschema/EventSchemaIntegrationConnector.java

```diff
@@ -0,0 +1,187 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright © 2021 Atruvia AG <opensource@atruvia.de> as contributor to the ODPi Egeria project. */
```

Comment:
We should check our contribution rules for the project. We've used a common odpi contributors copyright with all code going into open source so far.

We need to check this before merging in .

Reply:
This occurs in other files too - but I won't repeat.